plugins/available/base.plugin.bash

```diff
@@ -91,13 +91,14 @@ function pmdown ()
 
 function mkcd ()
 {
-    about 'make a directory and cd into it'
-    param 'path to create'
+    about 'make one or more directorys and cd into the lastest one'
```

Comment:
Some typos on this line:

- "directories" instead of "directorys"
- "last" instead of "lastest"